web/ui/src/Pools/poolsFactory.js

```diff
@@ -7,18 +7,13 @@
 
     angular.module('poolsFactory', []).
     factory("poolsFactory", ["$rootScope", "$q", "resourcesFactory", "$interval", "baseFactory",
-    function($rootScope, q, _resourcesFactory, $interval, BaseFactory){
+    function ($rootScope, q, _resourcesFactory, $interval, BaseFactory) {
 
         // share resourcesFactory throughout
         resourcesFactory = _resourcesFactory;
         $q = q;
 
         var newFactory = new BaseFactory(Pool, resourcesFactory.getPools);
-
-        // alias some stuff for ease of use
-        newFactory.poolList = newFactory.objArr;
-        newFactory.poolMap = newFactory.objMap;
```

Comment:
`poolList` is used in a number of other places (deploy wiz, service details, hosts map, hosts), so this needs to stay.